tests/python/gaia-ui-tests/gaiatest/apps/cost_control/regions/ftu_step3.py

```diff
@@ -65,9 +65,10 @@ def select_when_use_is_above_unit_and_value(self, unit, value):
         Wait(self.marionette).until(lambda m: int(data_limit_view.location['y']) == int(data_limit_view.size['height']))
 
     def tap_lets_go(self):
-        self.marionette.find_element(*self._go_button_locator).tap()
+        element = self.marionette.find_element(*self._go_button_locator)
+        element.tap()
+        Wait(self.marionette).until(expected.element_not_displayed(element))
         self.apps.switch_to_displayed_app()
-        self.wait_for_element_not_displayed(*self._ftu_frame_locator)
```

Comment:
Hrm, yeah, not sure why I changed this here, I'll change it back similar to what's already being used.